go/oasis-node/cmd/genesis/genesis.go

```diff
@@ -417,6 +421,8 @@ func AppendRootHashState(doc *genesis.Document, exports []string, l *logging.Log
 		RuntimeStates: make(map[common.Namespace]*registry.RuntimeGenesis),
 
 		Parameters: roothash.ConsensusParameters{
+			DebugDoNotSuspendRuntimes: viper.GetBool(cfgRoothashDebugDoNotSuspendRuntimes),
```

Comment:
Should these parameters require DontBlameOasis to set?

Reply:
They do, genesis sanity check will fail otherwise.